### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by sureshkrishnan83** [Badgerati/Pode#687] [Q&A]
Hi Mathew
I am reading through the documentation and on the basic section -  https://badgerati.github.io/Pode/Tutorials/Basics/
i am bit confused about the below part. 

then use ./Server.ps1 on the CLI.

CLI (start from CLI)

PS> Start-PodeServer -FilePath './File.ps1'

Should this be.    PS> Start-PodeServer -FilePath './server.ps1' ?

Could you please help me ? Thanks :)

**Comment by ili101** [comment]
Hi. As I understated he is showing 2 different ways to run the `File.ps1` code.
1. Is to create another `Server.ps1` file and run it. (More of a permanent solution)
2. Is to just write in the CLI `Start-PodeServer -FilePath './File.ps1'` (Kind of an extra flexible way to manually start a server maybe?)

**Reply by sureshkrishnan83** [reply]
ok make sense :) . thank you ..
